README.md

```diff
@@ -31,7 +31,7 @@ The API and app keys can be found at https://app.datadoghq.com/account/settings#
 Add the Datadog request handler to your middleware in `settings.py`.
 
 ```python
-MIDDLEWARE_CLASSES += ('datadog.middleware.DatadogMiddleware')
+MIDDLEWARE_CLASSES += ('django-datadog.middleware.DatadogMiddleware')
```

Comment:
Same here: Module names don't use a dash, because that can interfere with Python syntax in Python code.